### ISSUE TITLE
az iot du update stage fail

### ISSUE THREAD

**Issue by c20xh2** [Azure/azure-iot-cli-extension#595]
## Describe the bug
Unable to create an update with the CLI
**Command Name**
`az iot du update stage
Extension Name: azure-iot. Version: 0.18.0.`

**Errors:**
```
The command failed with an unexpected error. Here is the traceback:
No module named 'azure.storage.blob'
Traceback (most recent call last):
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\knack/cli.py", line 233, in invoke
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 663, in execute
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 726, in _run_jobs_serially
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 697, in _run_job
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/__init__.py", line 333, in __call__
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/command_operation.py", line 363, in handler
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/arm.py", line 429, in show_exception_handler
  File "D:\a\_work\1\s\build_scripts\windows\artifacts\cli\Lib\site-packages\azure/cli/core/commands/command_operation.py", line 361, in handler
  File "C:\Users\Phil\.azure\cliextensions\azure-iot\azext_iot\deviceupdate\commands_update.py", line 454, in stage_update
    from azext_iot.deviceupdate.providers.storage import StorageAccountManager
  File "C:\Users\Phil\.azure\cliextensions\azure-iot\azext_iot\deviceupdate\providers\storage.py", line 9, in <module>
    from azure.storage.blob import BlobServiceClient
ModuleNotFoundError: No module named 'azure.storage.blob'

```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az iot du update stage --account {} --instance {} --manifest-path {} --storage-account {} --storage-container {} --overwrite --then-import`

## Environment Summary
```
Windows-10-10.0.19044-SP0
Python 3.10.8
Installer: MSI

azure-cli 2.42.0

Extensions:
azure-iot 0.18.0
sentinel 0.1.1

Dependencies:
msal 1.20.0
azure-mgmt-resource 21.1.0b1
```

**Comment by digimaun** [comment]
@c20xh2 thanks reporting this. Please update your `azure-iot` extension to `0.18.1` and try again.

**Comment by digimaun** [comment]
This issue should be resolved in 0.18.1. If you still run into the same problem let us know and we can re-open the issue.
